openshift/tests-extension/test/olmv1-incompatible.go

```diff
@@ -37,14 +37,23 @@ import (
 )
 
 var _ = Describe("[sig-olmv1][OCPFeatureGate:NewOLM][Skipped:Disconnected] OLMv1 operator installation", func() {
+	var unique, nsName, ccName, rbName, opName string
 	BeforeEach(func() {
 		helpers.RequireOLMv1CapabilityOnOpenshift()
+		unique = rand.String(8)
+		nsName = "install-test-ns-" + unique
+		ccName = "install-test-cc-" + unique
+		rbName = "install-test-rb-" + unique
+		opName = "install-test-op-" + unique
```

Comment:
We only have one It right now, so this wouldn’t change behavior today.
But adding another It later without the fix could bite us. 
Let’s keep it consistent across the board so we don’t trip over this again.
Making this change now keeps things consistent and future-proof.